runner/runner.go

```diff
@@ -78,6 +78,24 @@ type Command struct {
 	SnapshotId string
 }
 
+func (c Command) String() string {
+	sfmt := "Command\n\tSnapshot ID:\t%s\n"
+	sargs := []interface{}{c.SnapshotId}
+
+	sfmt += "\t%v\n\tTimeout:\t%v\n"
+	sargs = append(sargs, c.Argv, c.Timeout)
+
+	if len(c.EnvVars) > 0 {
+		sfmt += "\tEnv:\n"
```

Comment:
Adding to sfmt is weird... maybe just do multiple Sprintfs so you don't need to store the fmt string at all?